deps/qt5-dependencies.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# vagrant-cookbook
+# Copyright (C) 2016 Pelagicore AB
+#      
+# Permission to use, copy, modify, and/or distribute this software for 
+# any purpose with or without fee is hereby granted, provided that the 
+# above copyright notice and this permission notice appear in all copies. 
+#  
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL 
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED  
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR 
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES 
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, 
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, 
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS 
+# SOFTWARE.
+#   
+# For further information see LICENSE
+# 
+#
+# Usage: qt5-dependencies.sh
+#
+# Installs the build dependencies for qt5
+#
+
+echo "Installing Qt5 related dependencies"
+
+function builddep {
+    packages="$@"
+
+    retval=100
+    count=0
+
+    if [[ "$retval" -ne "0" && $count -le 5 ]]; then
+        count=$count+1
+        DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confnew" --force-yes -fuy build-dep $packages
+        retval=$?
+    fi
+
+    if [[ "$retval" -ne "0" ]]; then
+        echo "Failed to install build-dep for " $packages
+        exit $retval
+    fi
+
+}
+
+function install {
+    packages="$@"
+
+    retval=100
+    count=0
+
+    if [[ "$retval" -ne "0" && $count -le 5 ]]; then
+        count=$count+1
+        DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confnew" --force-yes -fuy install $packages
+        retval=$?
+    fi
+
+    if [[ "$retval" -ne "0" ]]; then
+        echo "Failed to install " $packages
+        exit $retval
+    fi
+
+}
+
```

Comment:
The two functions `install` and `builddep` are very similar. Maybe it would be nice to merge them and handle their differences with parameters?